src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContractSet.cs

```diff
@@ -0,0 +1,528 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.Serialization
+{
+    using System;
+    using System.Xml;
+    using System.Xml.Schema;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Reflection.Emit;
+    using DataContractDictionary = System.Collections.Generic.Dictionary<System.Xml.XmlQualifiedName, DataContract>;
+    using System.Text;
```

Comment:
Thanks for finding this. Changed.

---
In reply to: [82037669](https://github.com/dotnet/corefx/pull/12378#discussion_r82037669) [](ancestors = 82037669)